src/kinetics/RxnRates.cpp

```diff
@@ -240,17 +228,19 @@ void PlogRate::validate(const std::string& equation)
     }
 }
 
-std::vector<std::pair<double, Arrhenius> > PlogRate::rates() const
+std::vector<std::pair<double, Arrhenius2> > PlogRate::rates() const
 {
-    warn_deprecated("PlogRate::rates", "Behavior to change after Cantera 2.6; "
-        "see getRates() for new behavior.");
     auto rateMap = getRates();
```

Comment:
In order to properly split `Arrhenius` into legacy and new behavior, this route is used again whenever you instantiate from XML and/or access the legacy objects. I'm not sure that this is the best place to make noisy warnings.

Reply:
I wonder if we need to have a way to do noisy warnings that are silenced if you're using CTI/XML input. Right now, we have a lot of functions and classes that are marked as deprecated but don't emit warnings for this reason.

Reply:
Maybe ... at the same time, it may be a good incentive to switch to YAML. I'd advocate for issuing warnings once `InterfaceKinetics` is done, and `Arrhenius2` is only used for XML/CTI.